org.elysium.parent/org.elysium.ui/src/org/elysium/ui/compiler/handlers/RecompileSelectedHandler.java

```diff
@@ -25,14 +27,17 @@
 import org.eclipse.util.ResourceUtils;
 import org.elysium.LilyPondConstants;
 import org.elysium.ui.compiler.LilyPondBuilder;
+import org.elysium.ui.markers.MarkerTypes;
 
 public class RecompileSelectedHandler extends AbstractHandler {
 
 	@Inject
 	private Provider<LilyPondBuilder> builder;
+	private boolean compileOutdatedOnly = false;
 
 	@Override
 	public Object execute(ExecutionEvent event) throws ExecutionException {
+		compileOutdatedOnly = event.getCommand().getId().endsWith("outdatedOnly");
```

Comment:
Please add a commandParameter to the command in the plugin.xml instead of deciding based on the command's name.

Reply:
I tried that. But I failed. Distinguishing the cases using command ids seemed a valid alternative. But I will check again, if I somehow can add a dummy parameter to the event.